autoscale_cloudroast/test_repo/autoscale/fixtures.py

```diff
@@ -132,7 +141,24 @@ def _set_up_clients():
 
 _rcv3_cloud_network = autoscale_config.rcv3_cloud_network
 
-autoscale_client, server_client, lbaas_client, rcv3_client = _set_up_clients()
+(autoscale_client, server_client, images_client,
+ lbaas_client, rcv3_client) = _set_up_clients()
+
+
+def fetch_image_ids(images_client):
```

Comment:
given that this hard-codes `Ubuntu`, either:

1. it should be called `fetch_ubuntu_image_ids`, or
2. it should be `def fetch_image_ids(images_client, name="ubuntu"):`

I guess this is a blocking call?